src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -8108,6 +8136,12 @@ namespace SurfaceGeometry {
                         state.dataSurface->SurfHasSurroundingSurfProperties(SurfLoop) = true;
                         state.dataSurface->SurfSurroundingSurfacesNum(SurfLoop) = state.dataSurface->SurfLocalEnvironment(Loop).SurroundingSurfsPtr;
                     }
+                    if (state.dataSurface->SurfLocalEnvironment(Loop).GroundSurfsPtr != 0) {
+                        state.dataSurface->IsSurfPropertyGndSurfacesDefined(SurfLoop) = true;
+                        state.dataSurface->UseSurfPropertyGndSurfTemp(SurfLoop) = true;
+                        state.dataSurface->UseSurfPropertyGndSurfRefl(SurfLoop) = true;
+                        state.dataSurface->GroundSurfsPropertyNum(SurfLoop) = state.dataSurface->SurfLocalEnvironment(Loop).GroundSurfsPtr;
+                    }
```

Comment:
if the input field `Ground Surfaces Object Name` is not blank, then sets the new surface variables to `true` and sets the index to the new object.